app/main.py

```diff
@@ -1,8 +1,23 @@
 class Person:
-    # write your code here
-    pass
+    people: dict = {}
```

Comment:
Using a class variable `people` to store instances of `Person` is unconventional and can lead to unexpected behavior, especially if multiple instances with the same name are created. Consider using an instance variable or another data structure to manage this.